app/templates/discord.hbs

```diff
@@ -19,32 +19,48 @@
           <p class="error__text">Something went wrong. Please try again.</p>
         </div>
       {{else if (eq this.linkStatus "not-linked")}}
+        <div class="linking__logo">
+          <img src="RDSLogo.png" alt="real_dev_squad" height="100px" width="100px" />
```

Comment:
why inline styling of the img?

Reply:
https://github.com/Real-Dev-Squad/website-my/pull/407#issuecomment-1554711369